test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PageWithNamespace_DesignTime.codegen.cs

```diff
@@ -1,3 +1,5 @@
+// <auto-generated/>
```

Comment:
It doesn't. I messed it up when I cherry picked from dev. Checksum generation behavior is different in dev and rel/15.3. I'll update this.